app/agents/unified/minimax_mcp_wrapper.py

```
@@ -0,0 +1,113 @@
+    return sys.executable
+
+
+def _get_real_args() -> List[str]:
+    raw = os.getenv("MINIMAX_MCP_REAL_ARGS")
+    if raw:
+        try:
+            data = json.loads(raw)
+            if isinstance(data, list):
+                return [str(item) for item in data]
+        except Exception:
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)